packages/ag-charts-community/src/chart/dom/theme.css

```diff
@@ -64,10 +64,16 @@
     --ag-charts-toolbar-button-padding-small: 1px;
 
     /* Popover */
-    --ag-charts-popover-background-color: #fafafb;
-    --ag-charts-popover-border: 1px solid var(--ag-charts-border-color);
-    --ag-charts-popover-active-color: var(--ag-active-color, #5090dc);
-    --ag-charts-popover-active-background-color: #e1f0fb;
+    --ag-charts-popover-background-color: var(--ag-charts-toolbar-background-color);
+    --ag-charts-popover-foreground-color: var(--ag-charts-toolbar-foreground-color);
+    --ag-charts-popover-border: var(--ag-charts-toolbar-border);
+    --ag-charts-popover-focus-color: var(--ag-charts-toolbar-focus-color);
+    --ag-charts-toolbar-hover-color: color-mix(
```

Comment:
Very minor, but we should move this to the 'toolbar' section of the file.

Reply:
Ah yes my bad. Updated

Reply:
👍